wrappers/MATLAB/CoolPropsWrapperTests.m

```diff
@@ -0,0 +1,115 @@
+classdef CoolPropsWrapperTests < matlab.unittest.TestCase
```

Comment:
Can you please rename to CoolPropWrapperTests everywhere?

Reply:
I will likely not have access to a computer for several days. Will be able to add the requested changes sometime later next week... If something changes I'll pibg you.

Reply:
ping*